src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -28,12 +36,23 @@ export const MyFieldAndBackground = () => {
 
   const person = personQuery.data;
   const personBackground = person?.background ?? {};
+  const personKeywords = person?.keywords ?? [];
+
+  const fieldQuery = useQuery({
+    enabled: !!debouncedSearchTerm,
+    queryKey: ['field', debouncedSearchTerm],
```

Comment:
`field` sier lite om hva dette er. Background er også fields på samme side feks
```suggestion
  const keywordsQuery = useQuery({
    enabled: !!debouncedSearchTerm,
    queryKey: ['keyword', debouncedSearchTerm],
```

Reply:
Var litt usikker på naminga her, siden field er fagfelt🤔 hva tenker du er best, `professionalField` eller `keywords`?

Reply:
siden det er `searchForKeywords()` som brukes her syns jeg `keywordsQuery` høres mest logisk ut. alternativt bør det også renames til `fetchProfessionalFields()` da, men da er det fortsatt ulikt fra pathen til APIet som er `/keywords`